### ISSUE TITLE
Narrator reading Nav items(button) twice in IE

### ISSUE THREAD

**Issue by rohitrewa** [OfficeDev/office-ui-fabric-react#9879]
IE version: 11.615.17763.0

## Codepen demonstrating the issue
 

This IE specific issue and IE does not support Codepen. But to repro you can create a button based nav. For creating a button based nav you need to provide onlcik handler

## Actual behavior:

On focusing button based Nav item Narrator reads the item twice. First time the content and second time the title.

The framework is setting the nav item name as title implicitly leading to this issue. On provding blank or null title framework still overrides it.

## Expected behavior:

The narrator should read each nav item only ones.

**Comment by ecraig12345** [comment]
Thanks for filing this! While it's not possible to write a codepen in IE, you can actually still view the debug mode of a saved codepen (created in another browser) in IE. So it would still be a lot easier for me to look into this if you could provide a codepen repro.

**Comment by ecraig12345** [comment]
Hmm actually just realized this one is assigned to @joschect not me 😆 but the same comment still applies.

**Comment by joschect** [comment]
@ecraig12345 yep! Thanks for getting this one started :)

**Comment by rohitrewa** [comment]
Sharing CodePen Link: https://codepen.io/rohit-gupta-the-vuer/pen/MNaMbq

**Comment by micahgodbolt** [comment]
Thanks for the link!

**Comment by rohitrewa** [comment]
Hey did we get the repro?

**Comment by rohitrewa** [comment]
@ecraig12345 I have a release blocker bug on my product due to this. Can you please update the current status on this.

**Comment by joschect** [comment]
@rohitrewa I was just able to reproduce this. Taking another look now.

**Comment by joschect** [comment]
@rohitrewa I need to confirm but I'm pretty sure this is working as intended. The reason this is occurring is that there is a title on the button so that a tooltip will display if a user hovers over it. The tooltip needs to be there in the case that there is a long name that gets truncated (see image)
![image](https://user-images.githubusercontent.com/20801176/62645794-cd2e9a00-b901-11e9-80cc-a43d710d22c7.png)

**Comment by rohitrewa** [comment]
A few points here

I do not see consistency in control’s behaviour for different versions of IE. Can you please refer shared Video in the mail. (Bad part is its happening in newer version)

Also in my opinion and as per the accessibility bug which I have a nav item should be read only once so that its not confusing to the listener. Please note that in other browsers the nav items are being read only once.

**Comment by joschect** [comment]
@rohitrewa I've confirmed that fabric is working as intended. There is a bug either in IE or Narrator. Theoretically the narrator should be smart enough to only read out the text once if they're identical and that's likely why this doesn't repro in edge. 

Here's a very similar issue that we fixed #7052 and you can see in the discussion about how it is necessary to have the tooltip/title there for truncated text. Please note in that particular bug there was extra hidden text, since that's removed the issue was fixed in edge. The main comment to look at is [here](https://github.com/OfficeDev/office-ui-fabric-react/issues/7052#issuecomment-438479995)
